hub-templates/basehub/values.yaml

```diff
@@ -318,6 +329,8 @@ jupyterhub:
             return resp
 
         c.JupyterHub.authenticator_class = CustomOAuthenticator
+        # TODO: find selected authenticator
+        c.CustomOAuthenticator.admin_users = get_config('staff.names.google')
```

Comment:
This is looking nice, but wondering how this will play with the `allowed_users` stuff...